SingularityUI/app/views/taskHealthcheckNotificationSubview.coffee

```diff
@@ -14,6 +14,7 @@ class taskHealthcheckNotificationSubview extends View
 
     render: =>
         return if not @model.synced
+        return if _.last(@model.attributes.taskUpdates).taskState is "TASK_FAILED" # This is unfortunately the only place last state is stored.
```

Comment:
We are pulling the task state in the same way in other places as well, no need for the comment here. Alternatively, if you want it cleaner here, you could add a line in `parse` on the model for the task and set an attribute called `currentState` using the code above. Then we have it for reference later on on each task model that we look at